src/project.rs

```diff
@@ -508,9 +508,11 @@ fn new_from_example_uses_example_and_target() {
     let proj = Project::from_example("hello").unwrap();
     assert_eq!(proj.root_dir, Path::new("examples/hello"));
     let output_dir = proj.output_dir.to_str_or_err().unwrap();
-    assert!(output_dir.starts_with("target/test_output/hello-"));
+    assert!(output_dir.starts_with("target/test_output/hello-") ||
+            output_dir.starts_with("target/test_output\\hello-"));
```

Comment:
Ah, now I see what you mean: That second line should be `target\\test_output\\hello-` with all back slashes. I can track it down. (Thought I got all those!)